tools/operator/testdata/barebones-test-input32.yml

```diff
@@ -1,7 +1,7 @@
 tls: true
```

Comment:
tls: true can be removed from test input file.

Reply:
ok